pkg/kds/context/testdata/full_sync/mesh-service/zone-1.golden.yaml

```diff
@@ -0,0 +1,86 @@
+# Mesh
+---
+creationTime: "0001-01-01T00:00:00Z"
+labels:
+  kuma.io/origin: global
+modificationTime: "0001-01-01T00:00:00Z"
+name: default
+type: Mesh
+
+# MeshService
+---
+creationTime: "0001-01-01T00:00:00Z"
+labels:
+  team: db-operators
+mesh: default
+modificationTime: "0001-01-01T00:00:00Z"
+name: redis-zone-1
+spec:
+  ports:
+  - appProtocol: tcp
+    port: 6739
+    targetPort: 6739
+  - appProtocol: tcp
+    name: some-port
+    port: 16739
+    targetPort: target-port-from-container
+  selector:
+    dataplaneTags:
+      app: redis
+      k8s.kuma.io/namespace: kuma-demo
+  state: Unavailable
+status:
+  addresses:
+  - hostname: redis.mesh
+    hostnameGeneratorRef:
+      coreName: kmy-hostname-generator
+    origin: HostnameGenerator
+  dataplaneProxies: {}
+  tls: {}
+  vips:
+  - ip: 10.0.1.1
+type: MeshService
+
+---
+creationTime: "0001-01-01T00:00:00Z"
+labels:
+  kuma.io/display-name: redis-zone-2
+  kuma.io/origin: global
+  kuma.io/zone: zone-2
+  team: db-operators
+mesh: default
+modificationTime: "0001-01-01T00:00:00Z"
+name: redis-zone-2-ff29vc4685c6f594
+spec:
+  ports:
+  - appProtocol: tcp
+    port: 6739
+    targetPort: 6739
+  - appProtocol: tcp
+    name: some-port
+    port: 16739
+    targetPort: target-port-from-container
+  selector:
+    dataplaneTags:
+      app: redis
+      k8s.kuma.io/namespace: kuma-demo
+  state: Unavailable
+status:
```

Comment:
shouldn't this status be removed?

Reply:
No, this resource was already on the zone and the point is to check if it wasn't removed by sync